scripts/tools/campaign_result_store.py

```
@@ -0,0 +1,212 @@
+        errors.extend(_validate_episode_parquet(parquet_path))
+    summary_path = output_dir / "summary.json"
+    if summary_path.is_file():
+        errors.extend(_validate_summary_file(summary_path))
+    return ResultStoreValidation(ok=not errors, errors=errors)
+
+
+def _validate_episode_parquet(parquet_path: Path) -> list[str]:
+    """Return validation errors for the episode Parquet surface."""
+    try:
+        episodes = pd.read_parquet(parquet_path)
+    except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)